ui/narrative/methods/run_QUAST_app/display.yaml

```diff
@@ -30,19 +30,19 @@ parameters :
 
 description : |
     <p>
-    This is a KBase wrapper for version 4.4 of the 
-    <a href="http://quast.sourceforge.net/" target="_blank">QUAST</a> 
-    assembly quality assessment tool. 
+    This is a KBase wrapper for version 4.4 of the
+    <a href="http://quast.sourceforge.net/" target="_blank">QUAST</a>
+    assembly quality assessment tool.
     </p>
 
-publications: 
+publications:
     -
         display-text: |
-            Alexey Gurevich, Vladislav Saveliev, Nikolay Vyahhi and Glenn Tesler, QUAST: quality assessment tool for genome assemblies, Bioinformatics (2013) 29 (8): 1072-1075. doi: 10.1093/bioinformatics/btt086 First published online: February 19, 2013
+            Gurevich A, Saveliev V, Vyahhi N, Tesler G. QUAST: quality assessment tool for genome assemblies. Bioinformatics. 2013;29: 1072–1075. doi:10.1093/bioinformatics/btt086
```

Comment:
Is the issue # left out intentionally?

Reply:
That's just the way Zotero generates the Public Library of Science citations. Going back through my other edited ones, it doesn't seem to have included issue numbers there either.